pom.xml

```diff
@@ -112,12 +112,14 @@
 		<benchmarkjar.name>benchmarks</benchmarkjar.name>
 		<javac.target>1.8</javac.target>
 		<slf4j.version>1.7.21</slf4j.version>
-		<log4j2.version>2.6.2</log4j2.version>
+		<log4j2.version>2.1</log4j2.version>
```

Comment:
When I ran it on the cluster I would get 
```
Exception in thread "main" java.lang.IllegalAccessError: tried to access method org.apache.logging.log4j.core.lookup.MapLookup.newMap(I)Ljava/util/HashMap; from class org.apache.logging.log4j.core.lookup.MainMapLookup
	at org.apache.logging.log4j.core.lookup.MainMapLookup.<clinit>(MainMapLookup.java:37)
```
It seems there's a conflict w/ the logging from Storm and this was my quick/lazy way to fix it. I'll try to dig into it some more.